README.md

```diff
@@ -66,7 +66,7 @@
 
 ## 🚀 View <a name="view"></a>
 
-- [Click here to view]https://htmlpreview.github.io/?https://github.com/luisrpavanelli/Hello-Microverse-Project/blob/main/index.html
+[Click here to view]: https://htmlpreview.github.io/?https://github.com/luisrpavanelli/Hello-Microverse-Project/blob/main/index.html
```

Comment:
Modify the line link, I have left a few comments that should be addressed before this gets merged